### DISCUSSION TITLE
Commentaires et idées

### DISCUSSION THREAD

**Discussion by JJHontebeyrie** [JJHontebeyrie/Companion#2] [General]
Bonjour à tous
Vous pouvez ici afficher vos remarques et commentaires au sujet du Companion.
Faites part également de vos idées

**Comment by willijm** [comment]
Bonjour,

pourquoi ne pas afficher un petit logo de la météo à  2 ou 4h a la place du niveau de la gauge d'affichage du niveau de luminosité?

Cordialement

**Reply by JJHontebeyrie** [reply]
Salut
Pour 2 raisons, l'afficheur est placé chez moi près de la fenêtre et en un coup d’œil, je connais la météo 😜, et pour faire ça, c'est assez complexe, il faut rechercher la météo et surtout prévoir autant de sprites que d'images météo possibles, c'est du boulot.
Mais c'est ouvert à tout développement, yapuka

**Comment by willijm** [comment]
Bonjour

Si tu veux j'ai ce bout de code
https://github.com/willijm/owm-esp32/blob/main/owm.ino pour recupere les
infos OwM

Cdt

Le dim. 22 janv. 2023 à 17:58, JJHontebeyrie ***@***.***> a
écrit :

**Comment by harry33160** [comment]
Bonjour,
Merci pour cette excellente application, il y a un bug sur l’affichage de la date, elle se met à jour à 23h chez moi.
D’autre part j’ai modifié ton .ino pour lire les informations des micros onduleurs emphase, si cela vous intéresse, je vous ferai passer mon .ino.
Cordialement

**Reply by JJHontebeyrie** [reply]
Ah enfin ! Le bug a été trouvé ! 
En fait je cherche toujours pourquoi 🙄
J'ai aussi des micro onduleurs Emphase, le code m'intéresse, après il faut voir comment intégrer tout ça...

**Reply by ledijonnais2** [reply]
Bonjour, J'ai aussi des micro onduleurs Emphase et le code m'intéresse ainsi que quelques explications pour la mise en place des bons fichiers. Merci

**Reply by harry33160** [reply]
Merci de me contacter avec votre e-mail, je peux pas envoyer le code avec github

**Comment by harry33160** [comment]
Bonjour,
Voici mon compagnon modifié. Je n’ai pas pris en compte vos modifications en 2.32.
Bien cordialement
Harry33160

**Comment by harry33160** [comment]
Bonjour,
J'ai intégré la version 2.33, bravo pour le clin d’œil poisson d'avril sur l'affichage météo

**Reply by JJHontebeyrie** [reply]
Par contre, je n'ai pas reçu le ino modifié avec enphase

**Reply by harry33160** [reply]
Bonsoir, 
Voilà mon companion modifié pour emphase, j'ai pris en compte vos modifs de la 3,36.
Cordialement
Harry
[CompanionV3.ino.zip](https://github.com/JJHontebeyrie/Companion/files/11151393/CompanionV3.ino.zip)

**Reply by JJHontebeyrie** [reply]
Cet ino est très intéressant, il faut que je le digère un peu et surtout que j’ai l’accès à ma passerelle, l’adresse ip me dit que je n’ai pas le droit, l’installateur n’a pas fait tout son boulot, je vais voir.
Mais cette version s’affranchie du routeur MSunPV, cela peut permettre aux enphasiens de pouvoir l’installerJe gère mon accès bloqué, je teste, et je reviens pour les nouvelles. José

**Comment by willijm** [comment]
Bonjour, 
Je suis entrain de regarder pour migrer le code vers vsc et platformio, pour simplifier le déploiement et pouvoir utiliser le data.

Que penses-tu de cette idée ?

**Reply by JJHontebeyrie** [reply]
Migrer vers Visual Studio, pourquoi pas ? Le problème est que une fois la version bien peaufinée, il ne faut pas que cela soit trop complexe à programmer pour que tout le monde puisse se le faire chez soi.

**Comment by harry33160** [comment]
Bonsoir,Je regarde demain pour le code , mais avant de vous l’envoyer avez vous des bonnes bases en c++ et arduino ide?MerciGILLES Le 29 avr. 2024 à 19:11, ledijonnais2 ***@***.***> a écrit :﻿
Bonjour, J'ai aussi des micro onduleurs Emphase et le code m'intéresse ainsi que quelques explications pour la mise en place des bons fichiers. Merci

**Comment by ledijonnais2** [comment]
Bonsoir,
  
Merci pour ce retour. Je pratique un peu Arduino et quelques bases en c++
  
Bonne soirée   
  
Frédéric   
  

  

  

  
  
  
  
  
>

**Reply by harry33160** [reply]
Voici le code modifié.
Bien respecter le fichier installation .pdf .
Modifier le fichier perso.h avec vos éléments.
ne pas mettre à jour la bibliothèque TFT_eSPI.h ma version 2.5.0.
Bonne compilation
cordialement
GILLES
[CompanionV250H1.zip](https://github.com/JJHontebeyrie/Companion/files/15167764/CompanionV250H1.zip)

**Comment by ledijonnais2** [comment]
Merci beaucoup,
  

  
Je m'occupe de tout cela.
  

  
Frédéric   
  

  

  
  
  
  
  
>
